libvmi/accessors.c

```diff
@@ -335,10 +335,10 @@ GSList* vmi_get_va_pages(vmi_instance_t vmi, addr_t dtb) {
 
 addr_t vmi_pagetable_lookup (vmi_instance_t vmi, addr_t dtb, addr_t vaddr)
```

Comment:
@bdpayne @smaresca I think this PR is close to getting merged, but we need some input on pushing the following change to the external API: making vmi_pagetable_lookup return status_t and use an output parameter for the return value: 

`status_t vmi_pagetable_lookup (vmi_instance_t vmi, addr_t dtb, addr_t vaddr, addr_t *paddr)`.